src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -862,5 +862,54 @@ public async Task BreakpointOnHiddenLineOfMethodWithNoNextVisibleLineShouldNotPa
             Task t = await Task.WhenAny(pause_task, Task.Delay(2000));
             Assert.True(t != pause_task, "Debugger unexpectedly paused");
         }
+
+        [Fact]
+        public async Task SimpleStepTest()
+        {
+            await SetBreakpoint("dotnet://debugger-test.dll/debugger-test.cs", 674, 0);
+
+            string expression = "window.setTimeout(function() { invoke_static_method ('[debugger-test] Foo:RunBart'); }, 1);";
+            await EvaluateAndCheck(
+                expression,
+                "dotnet://debugger-test.dll/debugger-test.cs", 674, 12,
+                "Bart");
+            await StepAndCheck(StepKind.Over, "dotnet://debugger-test.dll/debugger-test.cs", 677, 8, "Bart");
+            await StepAndCheck(StepKind.Over, "dotnet://debugger-test.dll/debugger-test.cs", 678, 4, "Bart");
+        }
+
+        [Fact]
+        public async Task StepAndEvaluate()
+        {
+            await SetBreakpoint("dotnet://debugger-test.dll/debugger-test.cs", 682, 0);
+
+            string expression = "window.setTimeout(function() { invoke_static_method ('[debugger-test] Foo:RunBart'); }, 1);";
```

Comment:
super nit: rename the variable to something else, or move the string to line 887. Just to avoid confusing it with the expression being evaluated in this test.

Reply:
done.